src/HTTPRequest.cpp

```diff
@@ -0,0 +1,125 @@
+#include "../include/HTTPRequest.hpp"
+#include <sstream>
+#include <string>
+
+HTTPRequest::HTTPRequest() {
+}
+
+HTTPRequest::~HTTPRequest() {
+}
+
+void	HTTPRequest::setMethodType(const std::string& requestLine) {
+	std::istringstream	stream_line(requestLine);
+	std::string word;
+	stream_line >> word;
+	if (word == "GET")
+		_methodType = HTTPMethod::GET;
+	else if (word == "POST")
+		_methodType = HTTPMethod::POST;
+	else if (word == "DELETE")
+		_methodType = HTTPMethod::DELETE;
+	else
+		_methodType = HTTPMethod::UNKNOWN;
```

Comment:
I would substitue this with a logger message e.g.:
Logger &logger = Logger::getInstance();
...
logger.log(WARNING, "Unkown method");